docs/proxy/atlas/metadata_configs.md

```diff
@@ -0,0 +1,210 @@
+# Popular Tables Configurations (Table, Metadata, User Relationship)
+
+In order to support different kind of statistics for Table (or even DataSet) we need to add
+a new entity `Metadata` and make a relationship between Metadata and DataSet.
+
+Also, in order to store User information and User's statistics and actions, we'll need a new User type entity, 
+which will be linked with Metadata as Readers, Owners, Managers etc.  
+
+We are not going to add custom attributes directly to a Table/DataSet entity, to make it backward compatible and clean of course.
+
+Following is the code to generate that relationship between Metadata, User and DataSet.
+
+```python
+    userReaderMetadata_DefDict = {
+        "entityDefs": [
+            {
+                "superTypes": ["Referenceable"],
+                "name": "User",
```

Comment:
Does it mean that ING will index user into atlas as well? This will leverage the new Amundsen people feature?

Reply:
@feng-tao to leverage the people feature and to support roles like managers, owners, readers etc. we've decided to create a user entity in atlas. The point here is that it's just a reference of a unique identifier of a user (corporate key, email etc), and the actual user detail will be fetched from ldap (at least in case of ING) but made generic enough so if other companies want to have all attributes of a user in atlas, they can still do that.